components/provisioner/internal/provisioning/input_converter_test.go

```diff
@@ -498,6 +501,11 @@ func fixKymaGraphQLConfigInput() *gqlschema.KymaConfigInput {
 					fixGQLConfigEntryInput("test.config.key2", "value2", util.BoolPtr(false)),
 				},
 			},
+			{
+				Component: rafterComponent,
+				Namespace: kymaSystemNamespace,
+				SourceURL: util.StringPtr(rafterSourceURL),
+			},
```

Comment:
Can we also add a component with the source URL to resolver integration tests? That way we will verify it works fine with DB,

Reply:
All right!